uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/absence-tab-custom-work-pattern-modal.component.spec.js

```diff
@@ -49,8 +49,12 @@
       });
 
       describe('init()', function () {
+        it('calls the valueOf with hrleaveandabsences_work_pattern_change_reason', function () {
```

Comment:
typo, it's `valuesOf`

however, I would recommend to write it in a more descriptive way from the business logic perspective

`fetches values of the Work Pattern Change Reason option group`

Reply:
Alright, thanks.